src/Components/Inspirations/Inspirations.scss

```diff
@@ -0,0 +1,99 @@
+
+$button-size: 8em;
+$button-color: black;
+$button-bg-color: rgba( $button-color, .4 );
+$button-stroke: $button-size/30;
+$arrow-base-size: $button-size/5;
+
+.play-button {
+  align-items: center;
+  border: $button-stroke solid $button-color;
+  border-radius: 50%;
+  box-shadow: 0 0 1px 0px rgba(255, 255, 255, 1);
+  cursor: pointer;
```

Comment:
We should move this line to the .play-button:hover block. It is not showing the pointer when the user hovers over the play arrow.

Reply:
all done